students/ghassan/lesson04/dict_lab.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+# dictionaries1
+cakes = {
+    'name': 'Chris',
+    'city': 'Seattle',
+    'cake': 'Chocolate'
+}
+
+print(cakes)
+cakes.pop('cake')
+print(cakes)
+cakes['fruit'] = 'Mango'
+print(cakes)
+print(cakes.keys())
+print(cakes.values())
+print('cake' in cakes.keys())
```

Comment:
This can be `'cake' in cakes` (`in` checks for existence of key in dict without needing to call `keys()`